### PR TITLE
Use `.inc.qhelp` extension for included help files

### PR DESCRIPTION
**Pull request by Marcono1234**
Follow-up for https://github.com/github/codeql/pull/5226#issuecomment-785750825. This pull request updates the `.qhelp` files only used in `include` by the other languages to use the extension `.inc.qhelp` as well.

### CODE REVIEW

File: docs/codeql/writing-codeql-queries/query-help-files.rst [hunk 1]
```diff
@@ -165,6 +165,7 @@ Query help inclusion
 ====================
 
 To reuse content between different help topics, you can store shared content in one query help file and then include it in a number of other query help files using the ``include`` element. The shared content can be stored either in the same directory as the including files, or in ``SEMMLE_DIST/docs/include``.
+When a query help file is only included by other help files but does not belong to a specific query, it should have the file extension ``.inc.qhelp``.
```
Comment (Marcono1234) [inline docs/codeql/writing-codeql-queries/query-help-files.rst:168]: Do you think it makes sense to include this? Any feedback is appreciated.

Comment (RasmusWL) [reply docs/codeql/writing-codeql-queries/query-help-files.rst:168]: I'm not on the documentation team, but this looks good to me :+1:

**Comment by aschackmull** [comment]
Looks like `csharp/ql/src/experimental/Security Features/Serialization/DefiningPotentiallyUnsafeXmlSerialization` also needs a rename.

**Comment by Marcono1234** [comment]
Ah right, though that file appears to be unused (#5274, unless I am overlooking something); should I simply delete it instead?

**Comment by aschackmull** [comment]
> Ah right, though that file appears to be unused (#5274, unless I am overlooking something); should I simply delete it instead?

@hvitved What do you say? Delete it?

**Comment by hvitved** [comment]
> @hvitved What do you say? Delete it?

It should actually be renamed to `DefiningPotentiallyUnsafeXmlSerializer.qhelp` to match the name of the query.

**Review by aschackmull — APPROVED** [review]

**Pull request merged**